dags/prio/processor.py

```diff
@@ -0,0 +1,330 @@
+    task_id="trigger_processor_b",
+    source_bucket=BUCKET_DATA_ADMIN,
+    source_object="staging/_SUCCESS",
+    destination_bucket=BUCKET_PRIVATE_B,
+    destination_object="raw/_SUCCESS",
+    google_cloud_storage_conn_id="google_cloud_prio_admin",
+    dag=dag,
+)
+
+# Initiate the processors by spinning up GKE clusters and running the main
+# processing loop. These SubDags are coordinated by a higher level graph, but
+# they can be decoupled and instead run on the same schedule. This structure is
+# done for convenience, since both processors are running on the same
+# infrastructure (but partitioned along project boundaries).
+
+processor_a = SubDagOperator(
+    subdag=kubernetes.container_subdag(
+        parent_dag_name=dag.dag_id,
+        child_dag_name="processor_a",
+        default_args=DEFAULT_ARGS,
+        server_id="a",
+        gcp_conn_id="google_cloud_prio_a",
+        service_account=SERVICE_ACCOUNT_A,
+        arguments=["processor/bin/process"],
+        env_vars={
+            "DATA_CONFIG": "/app/processor/config/content.json",
+            "SERVER_ID": "A",
+            "SHARED_SECRET": "{{ var.value.prio_shared_secret }}",
+            "PRIVATE_KEY_HEX": "{{ var.value.prio_private_key_hex_internal }}",
+            "PUBLIC_KEY_HEX_INTERNAL": "{{ var.value.prio_public_key_hex_internal }}",
+            "PUBLIC_KEY_HEX_EXTERNAL": "{{ var.value.prio_public_key_hex_external }}",
+            "BUCKET_INTERNAL_PRIVATE": BUCKET_PRIVATE_A,
+            "BUCKET_INTERNAL_SHARED": BUCKET_SHARED_A,
+            "BUCKET_EXTERNAL_SHARED": BUCKET_SHARED_B,
+            # 10 minutes of timeout
+            "RETRY_LIMIT": "60",
+            "RETRY_DELAY": "10",
+            "RETRY_BACKOFF_EXPONENT": "1",
+        },
+    ),
+    task_id="processor_a",
+    dag=dag,
+)
+
+processor_b = SubDagOperator(
+    subdag=kubernetes.container_subdag(
+        parent_dag_name=dag.dag_id,
+        child_dag_name="processor_b",
+        default_args=DEFAULT_ARGS,
+        server_id="b",
+        gcp_conn_id="google_cloud_prio_b",
+        service_account=SERVICE_ACCOUNT_B,
+        arguments=["processor/bin/process"],
+        env_vars={
+            "DATA_CONFIG": "/app/processor/config/content.json",
+            "SERVER_ID": "B",
+            "SHARED_SECRET": "{{ var.value.prio_shared_secret }}",
+            "PRIVATE_KEY_HEX": "{{ var.value.prio_private_key_hex_external }}",
+            "PUBLIC_KEY_HEX_INTERNAL": "{{ var.value.prio_public_key_hex_external }}",
+            "PUBLIC_KEY_HEX_EXTERNAL": "{{ var.value.prio_public_key_hex_internal }}",
+            "BUCKET_INTERNAL_PRIVATE": BUCKET_PRIVATE_B,
+            "BUCKET_INTERNAL_SHARED": BUCKET_SHARED_B,
+            "BUCKET_EXTERNAL_SHARED": BUCKET_SHARED_A,
+            # 10 minutes of time-out
+            "RETRY_LIMIT": "60",
+            "RETRY_DELAY": "10",
+            "RETRY_BACKOFF_EXPONENT": "1",
+        },
+    ),
+    task_id="processor_b",
+    dag=dag,
+)
+
+
```

Comment:
what are we inserting?

Reply:
Aggregates are being inserted into BigQuery.